Makefile

```
@@ -0,0 +1,64 @@
+YML = docker-compose.yml
```

this is not a good name, yml is a extension of files. Can you rename to DOCKER_COMPOSE

```suggestion
DOCKER_COMPOSE = docker-compose.yml
```